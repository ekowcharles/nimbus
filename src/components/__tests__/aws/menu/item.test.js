import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import MenuItem from '../../../aws/menu/item'

describe('Menutem,', () => {
  it('renders menu item', () => {
    const wrapper = shallow(
      <MenuItem
        checked={false}
        label="Hide Status"
      // update={[Function]}
      />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})