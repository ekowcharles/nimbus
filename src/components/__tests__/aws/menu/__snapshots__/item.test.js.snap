// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menutem, renders menu item 1`] = `
<div
  onClick={[Function]}
>
  <input
    checked={false}
    readOnly={true}
    type="checkbox"
  />
  <span>
    Hide Status
  </span>
</div>
`;
